### ISSUE TITLE
"Unknown game" title

### ISSUE THREAD

**Issue by Bobcat90** [Ertego/gog-galaxy-itch.io#5]
Hi, first of all thanks for making the itch.io integration, I was so happy to find it! I'm new to github so I'm sorry if I get something wrong. All of the games I installed on the itch.io client showed up in gog galaxy for me, but one of them appears as "Unknown game" as the title when it's supposed to be The Lady's Choice (https://seraphinite.itch.io/the-ladys-choice). The game does still launch properly though. When I click on the game page it comes up with "Sorry, we couldn't load the data. Retry", but that comes up for all the itch.io games too. It doesn't have an IGDB entry, but again it's not the only one. I've tried syncing integrations, disconnecting & reconnecting the itch.io integration and closing & reopening gog galaxy.

**Comment by lele-blue** [comment]
This could be related to #2, but I'll take a look at the issue :)

**Comment by EustaceCS** [comment]
Same for... sigh... 'Lewd Leaf Land', paid version.
Free demo is missing from Galaxy - paid version is 'Unknown game'

**Comment by Bobcat90** [comment]
I'm also having this issue for soko loco (https://sokpop.itch.io/soko-loco)

**Comment by lele-blue** [comment]
@Bobcat90 can you please attach the plugin logfile so I can see if the issue is caused by the plugin?

**Comment by Bobcat90** [comment]
@Ertego There's several log files so let me know if this isn't the right one: 
[plugin-itch-fd0effe6-77d9-42bc-999d-a5a5c870c2a4.log](https://github.com/Ertego/gog-galaxy-itch.io/files/4410283/plugin-itch-fd0effe6-77d9-42bc-999d-a5a5c870c2a4.log)

Edit: I might be misunderstanding, but it looks like the plugin can detect all the games you have in collections, but discards any that aren't installed. Does that mean that eventually it might be possible to show uninstalled Itch games in Galaxy?

**Comment by lele-blue** [comment]
@Bobcat90 yes thats the right one, and the plugin detects all games (at least i checked it for soko loco). So i'd say that the issue _might_ not be on the plugins side.

Showing all games will definitely be possible and implemented once #6 is resolved.

**Comment by emmanueljarri** [comment]
Hello, 
I have the same problem, one game appear as "Unknown game".
I can launch it so realized it's the game named "SUPER Cute Alien · Prologue ·"
with the dots as special characters. 
I attached the plugin log, I didn't notice anything special for this game : 
2020-04-04 23:26:20,266 - root - DEBUG - Building game 168078 (SUPER Cute Alien · Prologue ·)
2020-04-04 23:26:20,267 - root - DEBUG - Built 168078 (SUPER Cute Alien · Prologue ·)

It also happen that, in itch.io client, in my library of owned games, filtered with Installed and ordered by name, this exact game is the last of the list. No idea if it's related but worth mentionning I guess.  URL of this search: 
itch://library/owned?installed=true&sortBy=title&sortDir=default

![GameList](https://user-images.githubusercontent.com/52140864/78466349-27207300-76ce-11ea-8b31-06e58c153941.jpg)
![Plugin itch Unknown Game](https://user-images.githubusercontent.com/52140864/78466352-2b4c9080-76ce-11ea-84e1-418582c420f8.jpg)
[plugin-itch-fd0effe6-77d9-42bc-999d-a5a5c870c2a4.log](https://github.com/Ertego/gog-galaxy-itch.io/files/4432915/plugin-itch-fd0effe6-77d9-42bc-999d-a5a5c870c2a4.log)

**Comment by emmanueljarri** [comment]
Then I clicked Uninstall in itch client. (some files stay on the drive though), 
The "Unknown game" was removed.
I installed another game (Ripped Pants), it was imported correctly. 
Then reinstalled Super Cute Alien, and same, the game was imported as "Unknown game".
I decided to give a try to "The Lady's Choice" as it was problematic for @Bobcat90, and same for me, it was imported as "Unknown game", so I now have 2 "Unknown game" in my list. 
It looks like it has to do with some specific games, not the process of import. 
(And not with the ' character, as I also have "A Mortician's Tale", which is correctly imported.

**Comment by lele-blue** [comment]
Since the integration reports the games names correctly to gog, I'll flag this as a gog issue for now, since I don't think that a workaround is currently possible :/

**Comment by EzioTheDeadPoet** [comment]
Sadly I have the same issue with Night in the Woods. 😭

**Comment by lele-blue** [comment]
> Since the integration reports the games names correctly to gog, I'll flag this as a gog issue for now, since I don't think that a workaround is currently possible :/

Still at this, we have to wait for GOG to change the API so i can report name and cover :/

**Comment by lele-blue** [comment]
From the WebAPI fork of this integration:

> Note that many games on Itch are not listed in IGDB and therefore will show up as "Unknown Game". You can help fix this by adding them on [igdb](https://www.igdb.com/)

**Comment by EzioTheDeadPoet** [comment]
> From the WebAPI fork of this integration:
> 
> > Note that many games on Itch are not listed in IGDB and therefore will show up as "Unknown Game". You can help fix this by adding them on [igdb](https://www.igdb.com/)

But I am having this issue with games, that are definitely on there, since they are also on steam and searchable in GOG just the version of the game that gets added trough the itch.io plugin seems to have issues. 🤔

**Comment by tkrusterholz** [comment]
Same as @EzioTheDeadPoet, GOG cannot match Night In the Woods even though it is on IGDB.

I wish GOG would give us a "fix match" option so we can tell it which game it is, like Plex does.

**Comment by EzioTheDeadPoet** [comment]
Good Idea, honestly I will use the feature request function to mention it directly to the Team creating the launcher.

**Comment by bartok765** [comment]
Hi, sometimes happens in battlenet plugin. 
I've found that GOG people say it is bug: https://github.com/gogcom/galaxy-integrations-python-api/issues/95#issuecomment-617664206

> "Unknown Game" should never appear I would consider this as our backend bug. If matching fails Galaxy should display a title the plugin reports. 

They requested logs
